### PR TITLE
Package dependency added for host pc

### PR DESCRIPTION
**Pull request by mchauras-linux**
-> Package dependency added for host pc i.e. axios
-> vscode related files ignored in .gitignore file
-> nginx added around react to stop websocket errors
-> Dummy post request made to verify that we are able to send data from react to flask

### CODE REVIEW

**Comment by mchauras-linux** [comment]
@OjaswinM @masingh-linux Can you review this?

File: README.md [hunk 1]
```diff
@@ -10,3 +10,7 @@ docker-compose up
 http://localhost/
 
 ## Test 
+
+## Packages required
+
+gpiod
```
Comment (OjaswinM) [inline README.md:16]: We dont need this right now correct?

File: react-client/src/App.js [hunk 2]
```diff
@@ -16,6 +17,19 @@ function App() {
 			})
 	}, [])
 
+	const handleButtonClick = async () => {
+		try {
+			const dataToSend = { led: 'toggle' }; // Data to be sent
+			const response = await axios.post(url, dataToSend);
```
Comment (OjaswinM) [inline react-client/src/App.js:23]: maybe convert the fetch() call above to axios as well?

**Review by OjaswinM — COMMENTED** [review]
Hi Mukesh, looks good. Added some minor reviews/questions

File: README.md (hunk 1 above)
Comment (mchauras-linux) [reply README.md:16]: Actually this is something of pi os. But for debugging purposes we need this as its a gpio utility

File: react-client/src/App.js (hunk 2 above)
Comment (mchauras-linux) [reply react-client/src/App.js:25]: Done

**Review by OjaswinM — APPROVED** [review]
Moye moye

**Comment by OjaswinM** [comment]
Also I think we should enable the "Create a merge commit" waala option. Nhi to history won't be saved after merges

**Pull request merged**